File: dayTwo/JAVASCRIPT/level_two/task_four.js
const prompt = require("prompt-sync")();

let firstNumber = Number(prompt("Enter first number: "));
let secondNumber = Number(prompt("Enter second number: "));

if (firstNumber > secondNumber)
    console.log("The largest number is " + firstNumber);
else
    console.log("The largest number is " + secondNumber);

File: dayTwo/JAVA/levelTwo/TaskFour.java
import java.util.Scanner;

public class TaskFour {

    public static void main(String... args) {

        Scanner inputCollector = new Scanner(System.in);

        System.out.print("Enter first number: ");
        int firstNumber = inputCollector.nextInt();

        System.out.print("Enter second number: ");
        int secondNumber = inputCollector.nextInt();

        System.out.print("The largest number is ");
        if (firstNumber > secondNumber)
            System.out.println(firstNumber);
        else
            System.out.println(secondNumber);
    }
}
